src/main.rs

```diff
@@ -69,29 +69,34 @@ fn main() {
                      .and_then(|d| d.deserialize())
                      .unwrap_or_else(|e| e.exit());
 
-    println!("Using configuration at: {}", args.flag_conf);
-
-    let conf: Conf = Conf::load(&args.flag_conf).unwrap_or_else(|e| {
-        eprintln!("Error loading configuration: {}", e);
+    let rtd: Rtd = Rtd::from_args(args).unwrap_or_else(|err| {
+        eprintln!("Error loading configuration: {}", err);
         process::exit(1);
     });
-    if args.flag_verbose { println!("\n{}", conf.features); }
+
+    println!("Using configuration: {}", rtd.paths.conf.display());
+    if rtd.args.flag_verbose {
+        println!("\n[features]\n{}", rtd.conf.features);
```

Comment:
Missing a semicolon?

Reply:
Ah yes, indeed! Although thinking about it, it looks a bit weird having the colons there anyway, as well as the TOML-like square bracketed section headings, may get rid of them

Reply:
Fixed with f527dce